man/ocamlopt.1

```diff
@@ -753,6 +753,31 @@ Generate position-independent machine code.  This is the default.
 .B \-fno\-PIC
 Generate position-dependent machine code.
 
+.SH OPTIONS FOR FLAMBDA
```

Comment:
I think this whole section can appear before the backend-specific options, so all of this before Line 719 above (the "OPTIONS FOR THE IA32 ARCHITECTURE" section).

As a small tweak, possibly:
```suggestion
.SH OPTIONS FOR THE FLAMBDA MIDDLE-END
```